self-tests.yml

```diff
@@ -695,3 +718,15 @@ buildvariants:
     tasks:
       - name: ".agent .test"
       - name: ".cli .test"
+
+
+containers:
+  - name: evg-container
+    working_dir: /
+    image: "hadjri/evg-container-self-testing"
```

Comment:
This seems fine for the moment, but I believe we'll have to replace this with one of the security-vetted images, since we'll be requiring all users (including us) to run pre-vetted containers. Can we make a ticket to replace it?

Reply:
https://jira.mongodb.org/browse/EVG-19745